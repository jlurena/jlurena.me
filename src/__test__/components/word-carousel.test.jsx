import React from 'react';
import renderer from 'react-test-renderer';
import WordCarousel from '../../js/components/word-carousel';

test('<WordCarousel/>', () => {
  const words = ['Hello', 'World', 'Goodbye'];

  const component = renderer.create(
    <WordCarousel words={words} />
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
