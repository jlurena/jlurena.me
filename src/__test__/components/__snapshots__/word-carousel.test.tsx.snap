// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WordCarousel/> Rendering 1`] = `
Array [
  <span
    className="word"
  >
    Hello
  </span>,
  <span
    className="word"
  >
    World
  </span>,
  <span
    className="word"
  >
    Goodbye
  </span>,
]
`;
